app/token/answer.go

```diff
@@ -22,6 +23,15 @@ func (tt *Answer) UnmarshalString(raw string) error {
 // MarshalJSON marshals the answer token into JSON
 func (tt *Answer) MarshalJSON() ([]byte, error) { return marshalJSON(tt) }
 
+// MarshalString marshals the answer token into a string
+func (tt *Answer) MarshalString() (string, error) { return marshalString(tt) }
+
+// Sign returns a signed token as a string
+func (tt *Answer) Sign(privateKey *rsa.PrivateKey) (string, error) {
+	tt.PrivateKey = privateKey
```

Comment:
Much better for the service. 👍 
I am still not a fan of this way to inject the key into the string for marshaling (and so using this "hazy" introspection) but let's keep this for the moment.